cityscapesscripts/evaluation/evalPanopticSemanticLabeling.py

```diff
@@ -33,11 +33,14 @@
 # Image processing
 # Check if PIL is actually Pillow as expected
 try:
-    from PIL import PILLOW_VERSION
+    from PIL import PILLOW_VERSION  # for pillow version before 7.0.0
 except ImportError:
-    print("Please install the module 'Pillow' for image processing, e.g.")
-    print("pip install pillow")
-    sys.exit(-1)
+    try:
+        from PIL import __version__  # for pillow version after 7.0.0
```

Comment:
Technically `__version__` has been available since Pillow 3.4.0 (https://github.com/python-pillow/Pillow/pull/2027/commits/7bfb2527bb0b832f830f87c37bd165028118e442), released in October 2016:

```suggestion
        from PIL import __version__  # for pillow version since 3.4.0
```

Reply:
And a simpler fix could be to just try this:

```diff
-from PIL import PILLOW_VERSION
+from PIL import Image
```

Reply:
Thank you so much for your reply! It's really cool and helpful to know that `__version__` has been available since 3.4.0.
But I'm afraid just try `form PIL import Image` wouldn't serve its original purpose as "# Check if PIL is actually Pillow as expected"
https://github.com/mcordts/cityscapesScripts/blob/fd2dfeb03f9eceb7a6060485f154e8dbd78982a4/cityscapesscripts/preparation/json2labelImg.py#L23
Because deprecated PIL also has `Image` (see [Image.py](https://github.com/whatupdave/pil/blob/master/PIL/Image.py)), it will work just fine even if you are using "real" PIL which is outdated. 
Fortunately, since you kindly reminded me that `__version__` is available since 3.4.0, we can just universally check `pillow` with `from PIL import __version__`. Is that right?😬
> Technically `__version__` has been available since Pillow 3.4.0 ([python-pillow/Pillow@7bfb252](https://github.com/python-pillow/Pillow/commit/7bfb2527bb0b832f830f87c37bd165028118e442)), released in October 2016:

Reply:
Thank's for bringing this up and providing a fix. As pillow is modelled as a dependency in `setup.py`, I propose that we drop the try...catch completely. I also updated the `README.md` to instruct the user to use pip for installation: #104 

Fine with you?

Reply:
https://github.com/mcordts/cityscapesScripts/pull/104 looks much simpler. Thanks both!

Reply:
That's neat! Thanks!